src/react/controllers/map-controller.js

```diff
@@ -60,7 +60,7 @@ const propTypes = {
   // A map control instance to replace the default map controls
   // The object must expose one property: `events` as an array of subscribed
   // event names; and two methods: `setState(state)` and `handle(event)`
-  mapControls: PropTypes.shape({
+  controls: PropTypes.shape({
```

Comment:
> Are you proposing we change the interface in react-map-gl as well?

No strong opinions on that yet. Just saw an opportunity to make the different controllers more alike.